README.md

```diff
@@ -155,6 +155,10 @@ $ make \
 
 Check the details via [Test Suite for SSE2NEON](tests/README.md).
 
+### Optimization
+
+Misbehavior may exist when turning on optimization options. Developers should be aware of possible errors.
```

Comment:
This message was a bit misleading. You should summarize the issues reported by the developers as known considerations.

Reply:
More content has been added to explain the context